test/images/httpd-new/BASEIMAGE

```diff
@@ -4,4 +4,5 @@ linux/arm64=arm64v8/httpd:2.4.39-alpine
 linux/ppc64le=ppc64le/httpd:2.4.39-alpine
 linux/s390x=s390x/httpd:2.4.39-alpine
 windows/amd64/1809=REGISTRY/busybox:1.29-2-windows-amd64-1809
+windows/amd64/ltsc2019=REGISTRY/busybox:1.29-2-windows-amd64-ltsc2019
 windows/amd64/ltsc2022=REGISTRY/busybox:1.29-2-windows-amd64-ltsc2022
```

Comment:
These busybox versions will have to be updated to ``busybox-1.29-5-*`` as well, otherwise the image building process will fail. There's no ``busybox:1.29-2-windows-amd64-ltsc2019``. Same below.

Reply:
Updated.